apps/fluent-tester/src/E2E/pages/ButtonTestPage.ts

```diff
@@ -0,0 +1,10 @@
+import { BUTTON_TESTPAGE } from '../../RNTester/Consts';
+import { BasePage, By } from './BasePage';
+
+class ButtonTestPage extends BasePage {
+  get _testPage() {
+    return By(BUTTON_TESTPAGE);
+  }
+}
+
+export default new ButtonTestPage();
```

Comment:
One note here, exporting an instance of a class isn't a common pattern. Generally, you want to instantiate the class where you will use it.

Reply:
I was following RNW's style with this. Do you mean just removing the _new_ keyword

Reply:
When you export `new ButtonTestPage()`  you're instantiating the `ButtonTestPage` class. That's not common. Usually you want to export the class and not an instance of it. Idk why RNW decided to do it this way (maybe that's their way of enforcing a single instance). Not a big deal, if it makes sense to export the instance just add it to the README that you add so people know what's going on.

Reply:
Will do, I am creating the README in the next PR, it's almost done. I'll be sure to add a note of it there